src/backend/routes/login.js

```diff
@@ -1,28 +1,28 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const bcrypt = require('bcrypt');
-const queries = require('../db/queries');
+const bcrypt = require("bcrypt");
+const queries = require("../db/queries");
 
 /* GET home page. */
-router.post('/', async (req, res, next) => {
+router.post("/", async (req, res, next) => {
   const name = req.body.name;
   const inputPwd = req.body.pwd;
 
-  if(name !== 'bot'){
-    const databasePwd = await queries.findUser({name: name});
-    if(databasePwd.length < 1){
-        res.json({err: 'An account was not found for this user'});
-    }else{
-      bcrypt.compare(inputPwd, databasePwd[0].pwd, (err,result,body) => {
-        if(result){
-          res.json({name: name});
-        }else{
-          res.json({err: 'username or password incorrect'});
+  if (name !== "bot") {
+    const databasePwd = await queries.findUser({ name: name });
+    if (databasePwd.length < 1) {
+      res.json({ err: "An account was not found for this user" });
+    } else {
+      bcrypt.compare(inputPwd, databasePwd[0].pwd, (err, result, body) => {
+        if (result) {
+          res.json({ name: name });
```

Comment:
you could block access to your game endpoints for unauthenticated users but that would require you to return a session token or JWT here and check validity before allowing a request to change game state. there's drop in solutions for auth out there but if none of that is what you want them i would suggest removing login entirely because right now it doesn't do anything really

Reply:
As it is right now, I am using login to return a token to the front end which stores it in a global state which every component has access to. This token is really important for a few of the api endpoints. I was thinking about providing route protection using this global state in the router on the front end.

Reply:
all youre returning here is the name of the user which the frontend already knows since the user is logging in :upside_down_face: 

also doesn't look like you're using tokens in api requests at all: https://github.com/BrendanKelly94/uno-app/blob/master/src/frontend/utils/ApiEndpoint.js#L16

Reply:
It is pretty quirky the way I'm doing it... Once bcrypt is happy with the authentication., it just returns the name to say "hey i've logged in". You're right about the front end knowing what the name already is but I couldn't think of anything else to return to tell the front end that I logged in.

Also I think I've been using the term token incorrectly. I just meant I need to send the username in a json object to a post request.

I am going to look into session tokens in order to find a more elegant solution!  :)